wdae/wdae/measures_api/views.py

```diff
@@ -47,16 +47,19 @@ def post(self, request):
         data = request.data
         dataset_id = data["datasetId"]
         dataset = self.gpf_instance.get_wdae_wrapper(dataset_id)
-
         assert dataset is not None
         assert dataset.phenotype_data is not None
-        assert "measure" in data
+        if "measures" in data \
+                and not dataset.phenotype_data.has_measure(data["measures"]):
+            return Response(status=status.HTTP_400_BAD_REQUEST)
+        assert "measures" in data
```

Comment:
Why do we change "measure" to "measures"? What is the `gpfjs` used in this request? How has it been working till now?